packages/block-editor/src/components/block-list/block-mobile-floating-toolbar.native.scss

```diff
@@ -1,3 +1,6 @@
+$floating-toolbar-height: 44;
+$floating-toolbar-first-child-top: 32; // $floating-toolbar-height - padding
```

Comment:
this looks fragile depending on the padding, it looks like padding value is determined in gutenberg/packages/block-editor/src/components/block-list/style.native.scss
blockContainerFocused::padding-top

so how about we define a $block-focused-padding-top to a common place(we should be able to create src/_native.native.scss) and use this new constant in both block-mobile-floating-toolbar.native.scss and block-list/style.native.scss

so instead we can do this right?
$floating-toolbar-first-child-top: ($floating-toolbar-height - $block-focused-padding-top);

Reply:
Make sense! 👍

Reply:
I was going to mention, we could extract the shared styles into another file and import, but it seems you've already done that. :+1: